content/en/docs/hertz/tutorials/basic-feature/binding-and-validate.md

```diff
@@ -57,13 +57,24 @@ func main() {
 
 
 
-### Parameter binding priority
+### Parameter Binding Precedence
```

Comment:
Precedence is a more appropriate word for this context. Also see [Operator precedence
](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence)